packages/components/fields/text-field/README.md

```diff
@@ -41,31 +41,44 @@ export default Example;
+| `isRequired`           | `boolean`                                                                                             |          |           | Indicates if the value is required. Shows an the "required asterisk" if so.                                                                                                                                                                                       |
+| `touched`              | `boolean`                                                                                             |          |           | Indicates whether the field was touched. Errors will only be shown when the field was touched.                                                                                                                                                                    |
+| `autoComplete`         | `string`                                                                                              |          |           | Used as HTML autocomplete property                                                                                                                                                                                                                                |
+| `name`                 | `string`                                                                                              |          |           | Used as HTML name of the input component. property                                                                                                                                                                                                                |
+| `value`                | `string`                                                                                              |    ✅    |           | Value of the input component.                                                                                                                                                                                                                                     |
+| `onChange`             | `ChangeEventHandler`                                                                                  |          |           | Called with an event containing the new value. Required when input is not read only. Parent should pass it back as value.&#xA;<br />&#xA;Signature: `(event) => void`                                                                                             |
+| `onBlur`               | `FocusEventHandler`                                                                                   |    ✅    |           | Called when input is blurred&#xA;Signature: `(event) => void`                                                                                                                                                                                                     |
```

Comment:
Are `onBlur` and `onFocus` actually required?
It doesn't seem so based on the props.

Reply:
Good catch..  needed to re run generate readme. 
Thanks.